sound/soc/sof/ipc4-topology.c

```diff
@@ -3330,6 +3330,9 @@ static enum sof_tokens dai_token_list[] = {
 	SOF_COMP_EXT_TOKENS,
 };
 
+/* The module-copier uses the same tokens as the dai-copier */
+#define buffer_token_list dai_token_list
```

Comment:
@ujfalusi but the dai tokens includes stuff like dai_type and dai_index that arent applicable for the buffer either? What issue does this change really fix?

Reply:
@ranj063, you are right and I'm blind..
I think it was when I was looking at the PAUSE_SUPPORTED token support initially for IPC4 only. Adding it to the host copier's token list would also adds it to the buffer, which did not sounded right.
Listing a token for a set where it does not applies just sounded wrong.
Naming the host struct as common is not right as it is not a common set, it includes host only tokens, right?

The correct token list for a buffer would be:
```c
static enum sof_tokens buffer_token_list[] = {
	SOF_COMP_TOKENS,
	SOF_AUDIO_FMT_NUM_TOKENS,
	SOF_IN_AUDIO_FORMAT_TOKENS,
	SOF_OUT_AUDIO_FORMAT_TOKENS,
	SOF_COPIER_TOKENS,
	SOF_COMP_EXT_TOKENS,
};
```